apps/sample-app/src/components/PeopleAvatars.tsx

```diff
@@ -66,8 +66,7 @@ export const PeopleAvatarList: React.FC<AvatarProps> = (props: AvatarProps) => {
           People to Meet Today
         </Title3>
         <div>
-
-          {AvatarItemList.map(avatar => (
+          {AvatarItemList.map((avatar) => (
```

Comment:
?

Reply:
I think this is a leftover Prettier change? Not sure, but I didn't actually edit this file.